docs/proposals/fluentd-secure-forward.md

```diff
@@ -0,0 +1,35 @@
+# Allow Fluentd to secure forward to other Fluentd/Logstash (over ES-Copy)
```

Comment:
Ultimately it depends on what our fluentd needs to be functional.  At a minimum, it needs permissions to read from the host right?  Isn't that at least a serviceaccount and maybe some policy?  I can envision a customer wanting a simplified experience  for deploying just fluentd with the forwarding option and nothing else.  I would imagine they would prefer to run single command and/or template instead of having to execute various instructions.  Couldn't this be captured as a mode?  There is the caveat also of "I want everything else as well fluentd configured in THIS way..." which means that 'autodeploy' requires several switches.   Maybe its a matter of 'listing' the name of the plugins to add using a well defined name (e.g fluentd_options=es_copy,secure_forward)